packages/auth-providers/auth0/setup/src/setup.ts

```diff
@@ -3,7 +3,7 @@ import yargs from 'yargs'
 import { standardAuthBuilder } from '@redwoodjs/cli-helpers'
 
 export const command = 'auth0'
-export const description = 'Generate an auth configuration for Auth0'
+export const description = 'Set up an auth configuration for Auth0'
```

Comment:
I agree setup is better than generate here. 
For my npm-package setup I went with just "Setup auth for <provider>", because it really is more than just config. It's new packages, new files, codemods, env vars and maybe more.
But then I reverted that change because I didn't want to change all auth providers for that PR, and it'd look weird to have just one be different.

So if you want to shorten it to just "Set up auth for Auth0" I'm all for that. But leaving it like what you have here is good too. You decide

Reply:
I like "Set up auth for <provider>"—I'll make those changes